lib/bolt_ext/server_config.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'hocon'
+require 'bolt/error'
```

Comment:
Looks like this isn't used in this file?

Reply:
It is for `Bolt::ValidationError`.